BrainPortal/app/views/signups/new.html.erb

```diff
@@ -24,180 +24,167 @@
+        Institution Info
+      </legend>
+
+      <div class="two-column-grid">
+        <%= f.label(:institution, "* Name of Institution/Organization:") %>
+        <%= f.text_field :institution, :size => 60 %>
+
+
+        <%= f.label(:department, "* Department") %>
+        <%= f.text_field :department, :size => 60 %>
+
+
+        <%= f.label(:position, "* Current position or role:") %>
+        <%= f.select :position, options_for_select(
+            [["(select one)", ""], "Faculty",
+                                   "Postdoctoral",
+                                   "PhD. candidate",
+                                   "Masters student",
+                                   "Student",
+                                   "Researcher",
+                                   "Other"
+            ],
+            @signup.position)
         %>
-     </td>
-    </tr>
-
-    <tr>
-      <td colspan="2">&nbsp;</td>
-    </tr>
-
-    <tr>
-      <th><%= f.label(:comment, "Comments or special requests:") %></th>
-      <td>
-          <%= f.text_area :comment, :cols => 80, :rows => 5 %>
-          <br>
-          <small>
-                 Please tell us the name of the laboratory you work for, the name of its Principal Investigator (if not you), and if possible anyone<br>
-                 else you know in your lab who are already CBRAIN users. We'll use this information to create or add you to a <em>Site</em> within CBRAIN.
+
+        <%= f.label(:affiliation, "* Current affiliation:") %>
+        <%= f.select :affiliation, options_for_select(
+            [["(select one)", ""], "Academic",
+                                   "Private sector",
+                                   "Government",
+                                   "Non-profit",
+                                   "Other",
+            ],
+            @signup.affiliation)
+        %>
+
+
+        <%= f.label(:email, "* Institutional Email address:") %>
+        <%= f.text_field :email, :size => 60 %>
+
+        <p></p>
+        <p>
+          <small>Please supply the address of <strong class="warning"> your research institution.</strong>
+            Requests with non institutional address or email
+            will be <strong>ignored</strong>
           </small>
-      </td>
-    </tr>
+        </p>
+
+        <%= f.label(:street1, "Institution Street Address (line 1):") %>
+        <%= f.text_field :street1, :size => 60 %>
 
-    <% if current_user.present? && current_user.has_role?(:admin_user) %>
-      <tr>
-        <td colspan="2">&nbsp;</td>
-      </tr>
+        <%= f.label(:street2, "Institution Street Address (line 2):") %>
+        <%= f.text_field :street2, :size => 60 %>
 
-      <tr>
-        <th> <%= f.label(:admin_comment, "Private, admin-only comments:") %> </th>
-        <td> <%= f.text_area :admin_comment, :cols=>80, :rows => 5 %> </td>
-      </tr>
-    <% end %>
+        <%= f.label(:city, "* City of Institution:") %>
+        <%= f.text_field :city, :size => 30 %>
 
```

Comment:
Please adjust spruious blank lines. Some places you have 1, some places you have 2...

Reply:
ok